src/renderers/WebGLRenderer.js

```diff
@@ -1461,6 +1462,9 @@ class WebGLRenderer {
 
 			_this.clear();
 
+			const renderBackground = xr.enabled === false || xr.isPresenting === false || xr.hasDepthSensing() === false;
+			if ( renderBackground ) background.render( scene );
```

Comment:
I'm not sure I understand the motivation for this part of the change, why would XR presentation and depth sensing affect whether the background is drawn?

Reply:
I don't know the history of some of these decisions, unfortunately - just that it's already done elsewhere in WebGLRenderer to determine whether to draw the background. There's probably some clean up that should happen in the render function in this respect.

Reply:
Ok! I don't know much about that part but will approve the PR in any case, thanks again!